core/app/waste_item/domain/ports.py

```diff
@@ -12,6 +13,7 @@ def get(self, waste_item_id: str) -> WasteItem: ...
     def create(self, waste_item: WasteItem) -> WasteItem: ...
     def delete(self, waste_item_id: str) -> None: ...
     def list(self) -> list[WasteItem]: ...
+    def getMaterial(self, material_waste: str) -> StatsWasteItem: ...
 
```

Comment:
remember that the python standar is `kebab_case`, also, the name is not very descriptive

```suggestion
    def get_material_count(self, material_waste: str) -> StatsWasteItem: ...
```